component/src/test/java/io/siddhi/extension/store/mongodb/MongoDBConnectionSSLTest.java

```diff
@@ -126,8 +128,9 @@ private void dropCollection() {
     }
 
     private boolean doesCollectionExists() {
-        try (MongoClient mongoClient = new MongoClient(new MongoClientURI(uri + "?authMechanism=MONGODB-X509",
-                mongoClientOptionsBuilder))) {
+        MongoClientSettings mongoClientSettings = mongoClientSettingsBuilder.applyConnectionString(
+                new ConnectionString(uri + "?authMechanism=MONGODB-X509")).build();
```

Comment:
Don't we have an inbuilt method to denote that the authMechanism is `MONGODB-X509` - since this is a specific mongo client setting?

If we don't have shal we make `?authMechanism=MONGODB-X509` a constant?

Reply:
Updated